Src/AutoFixture.NUnit3.UnitTest/Scenario.cs

```diff
@@ -65,6 +65,33 @@ public void FreezeSecondParameterOnlyFreezesSubsequentParameters(Guid g1, [Froze
         }
 
         [Test, AutoData]
+        public void ModestCreatesParameterWithModestConstructor([Modest]MultiUnorderedConstructorType p)
+        {
+            Assert.True(string.IsNullOrEmpty(p.Text));
+            Assert.AreEqual(0, p.Number);
+        }
+
+        [Test, AutoData]
+        public void GreedyCreatesParameterWithGreedyConstructor([Greedy]MultiUnorderedConstructorType p)
+        {
+            Assert.False(string.IsNullOrEmpty(p.Text));
+            Assert.AreNotEqual(0, p.Number);
+        }
+
+        [Test, AutoData]
+        public void BothFrozenAndGreedyAttributesCanBeAppliedToSameParameter([Frozen][Greedy]MultiUnorderedConstructorType p1, MultiUnorderedConstructorType p2)
```

Comment:
In .NET, the order of attributes are not guaranteed. What happens if `[Frozen]` is evaluated and executed _before_ `[Greedy]`? Could that make the test fail?